src/components/catalog-search/catalog-search.controller.ts

```diff
@@ -56,6 +57,10 @@ export class CatalogSearchController implements angular.IController {
       this.$scope.$emit('open-overlay-panel', item);
     }
     this.ctrl.searchText = '';
+    this.ctrl.mobileSearchInputShown = false;
+    if (angular.isFunction(this.ctrl.searchToggleCallback)) {
```

Comment:
Nit: We typically use lodash when angular and lodash have the similar utilities (`_.isFunction`) unless there's a reason not to.